lib/ansible/plugins/lookup/items.py

```diff
@@ -19,6 +19,8 @@
       _terms:
         description: list of items
         required: True
+    notes:
+      - This lookup is deprecated in favor of "loop" and the "flatten" filter
```

Comment:
can the `flatten` filter be used to only flatten a single level?  Also, is it properly called `flatten` and not `flattened`?  If so, the docs should be updated: https://docs.ansible.com/ansible/devel/plugins/lookup/flattened.html

Reply:
The `lookup` is called `flattened` but the filter is called `flatten`.  It can specify levels, and we have documented examples at https://docs.ansible.com/ansible/latest/porting_guides/porting_guide_2.5.html#with-items

Reply:
great, thanks!